recipes-xfce/xserver-xfce-init/xserver-xfce-init-1.0/xserver-xfce.conf

```diff
@@ -1,2 +1,10 @@
+if [ "$1" = "start" ] && \
+	/usr/local/natinst/bin/nirtcfg --get section=SYSTEMSETTINGS,token=ui.enabled,value=False \
+	| grep -iq false
+then
+	echo "$0: Disabled in system configuration." >&2
+	exit 0
```

Comment:
Does this have the effect of exiting from the parent xserver-xfce process too? I assume that is the intent given the message.

Reply:
The intent of the original code was to not start the X stack (X server and Xfce desktop environment) if the "Enable Embedded UI" checkbox in MAX is not checked.

Reply:
Right. Since this file is invoked from [here](https://github.com/ni/meta-nilrt/blob/nilrt/master/dunfell/recipes-xfce/xserver-xfce-init/xserver-xfce-init-1.0/xserver-xfce#L8), I was wondering if this exit would cause the parent process to also finish or would it continue on to the remaining commands, which does not seem to be the intent.

Reply:
Do you have an update on this @chaitu236?

Reply:
Oh sorry I misunderstood the comment. Yes the exit would cause the parent to also exit and it wouldn't continue to line 26.